pkg/branding/brandedtext_test.go

```diff
@@ -0,0 +1,65 @@
+package branding
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	brandedProductNameRHACS    = "Red Hat Advanced Cluster Security for Kubernetes"
+	brandedProductNameStackrox = "StackRox"
+)
+
+var _ suite.SetupAllSuite = (*BrandedTextTestSuite)(nil)
+var _ suite.TearDownTestSuite = (*BrandedTextTestSuite)(nil)
+
+func TestBrandedText(t *testing.T) {
+	suite.Run(t, new(BrandedTextTestSuite))
+}
+
+type BrandedTextTestSuite struct {
+	suite.Suite
+	envIsolator *envisolator.EnvIsolator
+}
+
+func (s *BrandedTextTestSuite) SetupSuite() {
+	s.envIsolator = envisolator.NewEnvIsolator(s.T())
+}
+
+func (s *BrandedTextTestSuite) TearDownTest() {
+	s.envIsolator.RestoreAll()
+}
+
+func (s *BrandedTextTestSuite) TestGetBrandedProductName() {
+	tests := map[string]struct {
+		productBrandingEnv string
+		brandedProductName string
+	}{
+		"RHACS branding": {
+			productBrandingEnv: "RHACS_BRANDING",
+			brandedProductName: brandedProductNameRHACS,
+		},
+		"Stackrox branding": {
+			productBrandingEnv: "STACKROX_BRANDING",
+			brandedProductName: brandedProductNameStackrox,
+		},
+		"Default setting": {
+			productBrandingEnv: "ROX_PRODUCT_BRANDING",
+			brandedProductName: brandedProductNameStackrox,
+		},
```

Comment:
This test is confusing, as I wrote here https://github.com/stackrox/stackrox/pull/1282#discussion_r850288971
Could you either rename it or delete it, please?

Reply:
Okay, I deleted it